db/seeds.rb

```diff
@@ -1,4 +1,10 @@
-require 'csv'
-### Create new seed routine
-
-puts "Seed routine forthcoming"
+[
+  {country: 'Indonesia', title: 'Indonesia - Census of Agriculture 2003'},
+  {county: 'Libya', title: 'The effect of settler farming on indigenous agriculture: Evidence from Italian Libya'},
```

Comment:
This should be `country`. Small typo that will create the record but with no country name.

Reply:
Ahh, thanks for catching that!  Pushed a fix to the seed_data branch.